store/android/build.gradle

```diff
@@ -56,3 +56,5 @@ kotlin {
     coroutines 'enable'
   }
 }
+
+tasks.getByName("preBuild").dependsOn(":store:generateSqlDelightInterface")
```

Comment:
This is a bit troubling. Let's file a Kotlin bug about this. An `expectedBy` dependency needs to resolve sources through some task in the declared project so that code generators have a hook to run.

Reply:
it might not be needed anymore - let me verify

Reply:
yup, not needed anymore. Sweet. JS one might not be needed either.